### ISSUE TITLE
Add option to turn on past projections and display interval chart instead of line chart

### ISSUE THREAD

**Issue by ksewalk** [healthmap/mriids2#146]
Current mock up here:
https://invis.io/SAZKQNDXVHK#/447082616_outbreak_-_Covid_-_Deaths_-_Projected

Instead of showing a line for the past projections, they would like to use the same interval chart we are using for future projections.

Research interval chart options and update design to use an interval chart. if possible.

**Comment by ksewalk** [comment]
We would need the historic projection data in the weekly projections file

**Comment by fernandodonaire** [comment]
Hey @rachelchorney ! For this, could we just use the same `"LineChart"` chart type that we are using on the `CovidProjectionsChart`? So the `reportedDeaths` can be a column, the `projectedDeaths` can be another column (using the `50` confidence interval projections values) and the `2.5` and `97.5` interval projection values can be interval columns.

**Comment by rachelchorney** [comment]
@fernandodonaire Yes, I think they want to use the same chart so that sounds great!

**Comment by fernandodonaire** [comment]
> @fernandodonaire Yes, I think they want to use the same chart so that sounds great!

Great sounds good!

**Comment by fernandodonaire** [comment]
Hey @ksewalk , do you think Sangeeta would be able to provide another file that just has the Covid deaths historic projections?
